cmake/AstarteMQTTTransport.cmake

```diff
@@ -55,6 +55,12 @@ function(astarte_sdk_configure_mqtt_dependencies)
         set(URL_GIT_TAG v3.2.4)
         FetchContent_Declare(ada GIT_REPOSITORY ${URL_GIT_REPOSITORY} GIT_TAG ${URL_GIT_TAG})
         FetchContent_MakeAvailable(ada)
+
+        # Library to manage database operations
+        set(SQL_GIT_REPOSITORY https://github.com/SRombauts/SQLiteCpp.git)
+        set(SQL_GIT_TAG 3.3.2)
+        FetchContent_Declare(SQLiteCpp GIT_REPOSITORY ${SQL_GIT_REPOSITORY} GIT_TAG ${SQL_GIT_TAG})
+        FetchContent_MakeAvailable(SQLiteCpp)
```

Comment:
I would add this as a dependency of the sample since the sample is the only one using it for now. Then we can move it to the main library later on.